advanced/multi_agent_analyzer.py

```
@@ -72,7 +73,7 @@ def categorize(self, review_text):
             response_format={"type": "json_object"}
         )
         try:
-            result = json.loads(response.choices[0].message.content)
+            result = extract_json_from_text(response.choices[0].message.content)
         except json.JSONDecodeError as e:
             raise ValueError(f"Invalid JSON response from API: {e}") from e
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)